app/views/complaints/show.html.erb

```diff
@@ -7,17 +7,35 @@
   <div class="grid-row grid-gap">
     <div class="grid-col-4">
       <div class="summary-section">
-        <h2>Summary</h2>
-        <p>
-          <strong><%= @complaint.grantee %>:</strong>
-          <%= @complaint.summary %>
-        </p>
+        <div class="ct-section-divider">
+          <h2>Summary</h2>
+          <p>
+            <strong><%= @complaint.grantee %>:</strong>
+            <%= @complaint.summary %>
+          </p>
+        </div>
+        <% unless @tta_reports.empty? %>
+          <%= render "tta_report_list", :tta_reports => @tta_reports %> 
+        <% end %>
+        <div class="display-none" id="tta-activity-form">
+          <%= render "tta_activity_report_form", :complaint => @complaint %>
+        </div>
+        <div class="ct-section-divider">
+          <p>
+            <span class="text-middle">
+              <svg class="usa-icon text-primary" aria-hidden="true" focusable="false" role="img">
+                <use xlink:href="/assets/img/sprite.svg#add_circle"></use>
+              </svg>
+            </span>
+              <%= link_to "Add TTA Activity Report", "#", class: "ct-add-report-link", id: "js-add-tta-report" %>
```

Comment:
[question] Hmm, I don't love that this is a link that behaves more like a button. Do you think we should add any sort of aria indication so that it's clear that this isn't actually taking the user to a new page to add the activity report?

Reply:
we can make it a button ... but i made it a link because i also don't think it's acting really like a button (it's not posting anything.) it feels more like an on-page anchor link. (if i take your suggestion and focus on the input, it would feel more like that too i think).

i am very open to suggestions on what element to use to better indicate what's happening here to screen readers!

Reply:
oookay i did some more research and i do think link is correct here (it takes us to a different part of the page.) i did make it better but changing the href to the id of the form input, which also means it focuses without using javascript, so this seems like the right way to do it!

Reply:
okay having done more research and talked it over with @jduss4 i am reversing my thought -- since it shows/hides, it seems to be a button. i am going to add a button role to make sure this is read correctly by a screen reader and do a manual test of it.